client/my-sites/posts/post-list.jsx

```diff
@@ -277,7 +277,7 @@ var Posts = React.createClass( {
 				/>
 			);
 		} else {
-			if ( this.props.loading || ! this.props.sites.fetched ) {
+			if ( this.props.loading || ! this.props.haveSites ) {
```

Comment:
`haveSites` seems ok to me in this context. It's only there because of the case where `props.loading` is false, but only because we don't have sites yet.